### PR TITLE
[NUI] Make AnchorClicked event public

### PR DESCRIPTION
**Pull request by wonrst**
Signed-off-by: Bowon Ryu <bowon.ryu@samsung.com>

### Description of Change ###
Public open API list

+ AnchorClickedEventArgs
  + string Href {get; set;}

TextLabel
+ event EventHandler<AnchorClickedEventArgs> AnchorClicked

TextField
+ event EventHandler<AnchorClickedEventArgs> AnchorClicked

TextEditor
+ event EventHandler<AnchorClickedEventArgs> AnchorClicked

### API Changes ###

 - ACR: TCSACR-415
 - https://code.sec.samsung.net/jira/browse/TCSACR-415

### CODE REVIEW

**Review by hyunjushin — APPROVED** [review]

File: src/Tizen.NUI/src/public/BaseComponents/TextEvent.cs [hunk 1]
```diff
@@ -23,15 +23,13 @@ namespace Tizen.NUI.BaseComponents
     /// <summary>
     /// AnchorClickedEventArgs is a class to record anchor click event arguments which will sent to user.
```
Comment (wiertel) [inline src/Tizen.NUI/src/public/BaseComponents/TextEvent.cs:24]:
```suggestion
    /// AnchorClickedEventArgs is a class to record anchor click event arguments which will be sent to user.
```

File: src/Tizen.NUI/src/public/BaseComponents/TextEvent.cs [hunk 2]
```diff
@@ -23,15 +23,13 @@ namespace Tizen.NUI.BaseComponents
...
     /// </summary>
-    /// This will be public opened in tizen_6.5 after ACR done. Before ACR, need to be hidden as inhouse API.
-    [EditorBrowsable(EditorBrowsableState.Never)]
+    /// <since_tizen> 9 </since_tizen>
     public class AnchorClickedEventArgs : EventArgs
     {
         /// <summary>
         /// Anchor href.
```
Comment (wiertel) [inline src/Tizen.NUI/src/public/BaseComponents/TextEvent.cs:30]: It should be enough as almost everybody know what href is, but consider adding expanded abbreviation in parenthesis.

File: src/Tizen.NUI/src/public/BaseComponents/TextEvent.cs (hunk 1 above)
Comment (wonrst) [reply src/Tizen.NUI/src/public/BaseComponents/TextEvent.cs]: Done

File: src/Tizen.NUI/src/public/BaseComponents/TextEvent.cs (hunk 2 above)
Comment (wonrst) [reply src/Tizen.NUI/src/public/BaseComponents/TextEvent.cs]: Done

**Comment by dongsug-song** [comment]
https://code.sec.samsung.net/jira/browse/TCSACR-415 has been accepted, so merge this now.

**Pull request merged**